skrub/_drop_column_if_null.py

```diff
@@ -21,12 +35,35 @@ def fit_transform(self, column, y=None):
 
         Returns
         -------
-            The input column, or an empty list if the column contains only null values.
+            The input column, or an empty list if the column is selected to be
+            dropped depending on the threshold.
         """
         del y
 
-        self.drop_ = sbd.is_all_null(column)
+        if self.threshold is not None:
+            if (
+                not (
+                    isinstance(self.threshold, float) or isinstance(self.threshold, int)
```

Comment:
you can use 
```python
import numbers
isinstance(self.threshold, numbers.Number)
```
instead.

it will work with numpy numeric types whereas

```
>>> isinstance(np.float32(0.5), float)
False
```

Reply:
Thanks, I knew that check didn't look right, but I wasn't sure how to write it properly

Reply:
you're welcome! also next time you want a disjunction of isinstance checks you can put the types in a tuple
```
>>> isinstance(2, (str, int))
True
>>> isinstance(2.5, (str, int))
False
```